src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -81,7 +88,15 @@ const revisionRow = style({
       alignItems: 'center',
       justifyContent: 'center',
       flexDirection: 'column',
-      padding: '10px 0px',
+      padding: '10px 15px',
+    },
+    'mann-witney-browser-name': {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexDirection: 'column',
+      padding: '10px 15px',
+      maxWidth: '80px',
```

Comment:
Can you conditionally set the maxWidth here for manwhitney test version below vs here?